### ISSUE TITLE
FAILED IMPORT INTO DATABASE

### ISSUE THREAD

**Issue by Guillermopl** [ESX-Org/esx_skin#68]
Error: Error Code:1146. Table es_extended.users doesn´t exist

**Comment by MonoChirito** [comment]
yeah that means in your database is no table named users. import the sql from es extended and thats the fix
